Lexic.txt

```diff
@@ -0,0 +1,29 @@
+a.Special symbols, representing:
+
+<operator> :== + | - | * | / | <- | < | <= | = | >=
+<separator> :== [ | ] | { | } | : | ; | space
+<reserved_word> :== var | integer | boolean | char | string | read | print | if | else | loop_until
+
+
+b. identifiers
+
+<letter> ::= a | b | ... | z | A | B | ... | Z
+<digit> ::= 0 | 1 | ... | 9
+<id> ::= <letter> { <letter> | <digit> | _ }
+
+c. constants
+1.integer - rule:
+	<noconst> ::= +<no>|-<no>|<no>
+    <no> ::= <digit><no>
+
+2.character
+
+    <character>::=<letter>|<digit>
```

Comment:
Din câte am înțeles, char ar trebui să conțină și apostrofuri, deci ar trebui să folosești alte reguli pentru <letter> și <digit> în care să pui fiecare caracter între apostrofuri.